scripts/custom-screens/CustomDataDisplayComponents.tsx

```diff
@@ -1,52 +1,60 @@
-import {registerScreen} from "@haulmont/jmix-react-web";
-import styles from "../App.module.css";
-import {Card} from "antd";
+import { registerScreen, Label } from "@haulmont/jmix-react-web";
+import { Card, Space } from "antd";
 import { EntityHierarchyTree } from "@haulmont/jmix-react-antd";
+import styles from "../App.module.css";
 
-const ROUTING_PATH = "/CustomDataDisplayComponents";
+const ROUTING_PATH = "/customDataDisplayComponents";
 
 const CustomDataDisplayComponents = () => {
-  return (
-    <Card className={styles.narrowLayout} title="EntityHierarchyTree">
-      <EntityHierarchyTree
-          hierarchyProperty="parent"
-          items={[
-            {
-              id: "A",
-              _instanceName: "Node A",
-          }, {
-              id: "A1",
-              parent: "A",
-              _instanceName: "Node A1",
-          }, {
-              id: "A2",
-              parent: "A",
-              _instanceName: "Node A2",
-          }, {
-              id: "A21",
-              parent: "A2",
-              _instanceName: "Node A21",
-          }, {
-              id: "B",
-              _instanceName: "Node B",
-          }, {
-              id: "B1",
-              parent: "B",
-              _instanceName: "Node B1",
-          }]}
-      />
-    </Card>
-  )
+    return (
+        <Space direction="vertical" style={{width: "100%"}} className={styles.narrowLayout}>
```

Comment:
`style={{width: "100%"}}` is used for set width and `className={styles.narrowLayout}` is used for set width too.
Maybe need to use an addition component?
```suggestion
        <div className={styles.narrowLayout}>
           <Space direction="vertical" style={{width: "100%"}}>
```